scripts/juno/snapshot/chain-node.sh

```diff
@@ -34,9 +34,11 @@
                 sed -i "/^minimum-gas-prices = .*/ s//minimum-gas-prices = \"$MINIMUM_GAS_PRICE\"/" $APP_FILE
                 sed -i "/^snapshot-interval = .*/ s//snapshot-interval = 1000/" $APP_FILE
 
-                wget -O - http://repository.activenodes.io/snapshots/juno-1_2022-03-01.tar.gz | tar -xz -C $CHAIN_DIR
+                LATEST=$(curl -s https://snapshots2.polkachu.com/snapshots/ | grep -oE 'juno/juno_.*.tar.lz4' | cut -f 1 -d '<' | head -1)
```

Comment:
I believe this was meant to be updated in the sentry.sh script for juno.

Reply:
Are you saying i should remove snapshots from the chain-node.sh?

Reply:
We should include this code in `scripts/juno/snapshot/sentry.sh` since the polkachu snapshots are mainly for validators. If we can find snapshots for juno with more history, then we should use that here, otherwise the polkachu snapshot is fine here also.